src/arrayMethodReduce.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result;
```

Comment:
You can assign `initialValue` here from the start and avoid `else` later
```suggestion
    let result = initialValue;
```